### PR TITLE
feat(operator_brief): Operator Brief seam v0 (first ontology-native seam)

### PR DESCRIPTION
**Pull request by AmitabhainArunachala**
## Summary

Implements the first ontology-native Operator Brief seam end-to-end per [`docs/plans/ONTOLOGY_NATIVE_OPERATOR_BRIEF_MASTER_SPEC.md`](docs/plans/ONTOLOGY_NATIVE_OPERATOR_BRIEF_MASTER_SPEC.md), then wires the next narrow runtime bridge from [`docs/plans/NEXT_10_SUBSTRATE_TODO.md`](docs/plans/NEXT_10_SUBSTRATE_TODO.md): cited runtime sources and `artifact_records` mirroring.

One successful `run_once()` tick now creates the required chain:

- `ActionProposal`
- gate decisions for `CONSENT`, `BHED_GNAN`, `STEELMAN`, and `DOGMA_DRIFT`
- any upstream top-level TelosGatekeeper BLOCK outside those four gates, recorded as an additional fail-closed `GateDecisionRecord`
- witness records for tick start, each gate, and materialisation
- existing `VentureCell` container for `cause:operator_brief_v0` instead of a new Movement schema
- `KnowledgeArtifact` with `subtype=operator_brief`
- artifact path and SHA-256 digest
- matching `RuntimeStateStore.artifact_records` row with the same artifact id/path/hash
- `Outcome`
- `ValueEvent`
- `Contribution`

No new bridges, routers, ledgers, memory stores, Go/Rust, dashboard work, commerce models, Dharma Radar work, or Movement engine are introduced. This moves substrate-nativeness by one user-visible seam only.

## Files changed

- `dharma_swarm/operator_brief/__init__.py` — package marker for the new seam subdirectory.
- `dharma_swarm/operator_brief/insight_brief.py` — public seam module with `run_once()` and `cron_run()`; kept under the 1,000-line module budget.
- `dharma_swarm/operator_brief/persistence.py` — artifact file materialisation and runtime `artifact_records` mirroring.
- `dharma_swarm/operator_brief/types.py` — seam constants and internal data carriers.
- `dharma_swarm/cron_runner.py` — dispatch branch for the `operator_brief` handler.
- `cron_jobs.json` — default-off `operator_brief` cron entry gated by `DHARMA_OPERATOR_BRIEF_ENABLED`.
- `tests/test_operator_brief_insight_brief.py` — focused seam coverage, including temp runtime-state sources, artifact-record mirroring, fail-closed cases, upstream gatekeeper block preservation, cron no-op behavior, and static no-raw-bypass checks across the seam modules.

## Behavior verified

- Uses existing `OntologyRegistry` / `OntologyObj` primitives and existing ontology types: `KnowledgeArtifact`, `WitnessLog`, `ActionProposal`, `GateDecisionRecord`, `Outcome`, `ValueEvent`, `Contribution`, `AgentIdentity`, and `VentureCell`.
- Runtime input is read from canonical `RuntimeStateStore` `session_events` / `memory_facts` when no explicit test payload is supplied.
- The real cron/default runtime store choice is explicit via `_default_runtime_state()`; tests still inject temp runtime DBs and do not write to real `~/.dharma`.
- Missing runtime sources or missing cited fact ids do not produce a valid brief.
- Required gates are evaluated in order. Any BLOCK/WARN on the required gates is treated as fail-closed for v0 and prevents artifact materialisation.
- Top-level `TelosGatekeeper` BLOCK decisions from non-required gates such as `AHIMSA` or `SATYA` are now preserved as additional blocking gate records and prevent artifact materialisation.
- `has_gate_decision` remains `ONE_TO_ONE`; the first gate decision uses the typed link and all gate decisions carry `proposal_id`, matching the existing `telic_seam.py` workaround pattern without schema changes.
- `STEELMAN` and `DOGMA_DRIFT` are locally enforced because the upstream gates are broader pattern checks; the local checks are documented in the module and covered by tests.
- Cron is default-off. With `DHARMA_OPERATOR_BRIEF_ENABLED` unset, direct `cron_run()` and `execute_cron_job()` dispatch do not call `run_once()` and do not mutate ontology/artifact state.
- No raw JSONL append or raw `open(..., 'w'/'a'/'x')` bypass was introduced.
- Artifact row insertion happens only after the markdown file and SHA-256 digest are valid; write/materialisation failures do not leave an incomplete `KnowledgeArtifact` row that could poison idempotency.
- Module-budget remains satisfied without grandfathering: `insight_brief.py` is now 950 lines, `persistence.py` is 278 lines, and `types.py` is 67 lines.

## Live tick

A manual enabled tick was run locally after the runtime wiring:

```bash
DHARMA_OPERATOR_BRIEF_ENABLED=1 python -c "from dharma_swarm.operator_brief.insight_brief import cron_run; print(cron_run({'id':'operator_brief'}))"
```

Result:

- `status=ran`
- `outcome=success`
- `artifact_id=e4bccf8689cd4de5`
- `runtime_artifact_id=e4bccf8689cd4de5`
- artifact path: `/Users/dhyana/.dharma/artifacts/operator_brief/2026-05-01/e4bccf8689cd4de5.md`
- runtime `artifact_records` row exists with matching SHA-256 digest

The feature flag remains default-off in `cron_jobs.json`; this was a one-shot explicit env-enabled tick.

## Final verification on head `b66ffb9`

```bash
python -m pytest tests/test_operator_brief_insight_brief.py -v
# 15 passed, 1 warning

python -m pytest tests/test_telic_seam.py tests/test_telos_gates.py tests/test_ontology.py -q
# 145 passed, 1 warning

python -m pytest tests/test_cron_runner.py tests/test_cron_scheduler.py tests/test_cron_daemon.py -q
# 41 passed, 1 warning

python -m compileall -q dharma_swarm tests
# passed

python scripts/governance/check_module_budget.py --base-ref origin/main --head-ref HEAD
# passed

python scripts/governance/check_test_hygiene.py
# passed; reports only the pre-existing tests/test_full_loop.py known warning
```

Regression reproduced before the fix: a SATYA credential marker in the generated brief returned `success` and wrote an artifact. After `b66ffb9`, the same scenario returns `failed_gate:SATYA` with no artifact.

The pytest warning is the existing `PytestConfigWarning: Unknown config option: timeout`; no test failed. Commit hooks also passed, including test hygiene, gitleaks, local Semgrep, whitespace, EOF, merge-conflict, and large-file checks.

## Known limitations / next steps

- `memory_facts` are supported as citations, but the current live runtime DB had zero rows there; the live tick cited `session_events` only.
- Failure paths emit an `Outcome` and zero-value `ValueEvent` so later watchdogs can distinguish "ran and failed" from "did not run". `Contribution` remains success-only.
- `STEELMAN` local enforcement checks the generated brief body for a counterargument marker; it does not replace a future stronger upstream gate.
- The next step after merge is `NEXT_10_SUBSTRATE_TODO` item 8: add the `LEDGER_WATCHER` check for operator-brief cron ticks that fail to produce `KnowledgeArtifact` rows.

## Substrate-nativeness estimate

- Before this PR: ~10-15% per `reports/audit/end_to_end/000_MASTER_COHERENCE_SYNTHESIS.md`.
- After this PR: ~10-15% plus one user-visible seam at 100% native, behind a default-off flag.

## Links

- Master spec: [`docs/plans/ONTOLOGY_NATIVE_OPERATOR_BRIEF_MASTER_SPEC.md`](docs/plans/ONTOLOGY_NATIVE_OPERATOR_BRIEF_MASTER_SPEC.md)
- Handoff: [`docs/plans/HANDOFF_ONTOLOGY_NATIVE_OPERATOR_BRIEF.md`](docs/plans/HANDOFF_ONTOLOGY_NATIVE_OPERATOR_BRIEF.md)
- Build entrypoint: [`docs/governance/BUILD_SESSION_ENTRYPOINT.md`](docs/governance/BUILD_SESSION_ENTRYPOINT.md)
- Next-10 todo: [`docs/plans/NEXT_10_SUBSTRATE_TODO.md`](docs/plans/NEXT_10_SUBSTRATE_TODO.md)

### CODE REVIEW

**Comment by AmitabhainArunachala** [comment]
## Handoff

- verdict: ready at code/test level; GitHub still reports draft because the automated mark-ready mutation failed with a connector GraphQL response-field error and local `gh` has bad credentials.
- files changed: `dharma_swarm/operator_brief/insight_brief.py`, `tests/test_operator_brief_insight_brief.py` on top of the existing PR files.
- tests run: `python -m pytest tests/test_operator_brief_insight_brief.py -v` (12 passed); `python -m pytest tests/test_telic_seam.py tests/test_telos_gates.py tests/test_ontology.py -q` (145 passed); `python -m pytest tests/test_cron_runner.py tests/test_cron_scheduler.py tests/test_cron_daemon.py -q` (41 passed); `python -m compileall -q dharma_swarm tests` (passed).
- remaining risks: `_collect_input` still has no live `session_events` / `memory_facts` wiring; feature flag remains off; PR needs manual Ready for review toggle if GitHub still shows draft.
- exact next step after merge: do `NEXT_10_SUBSTRATE_TODO` item 6: enable one operator profile, run one live tick, and capture `reports/witness/<date>-operator-brief-first-tick.md`.

**Comment by AmitabhainArunachala** [comment]
## PR #57 handoff

Verdict: ready in code; GitHub still shows draft because both ready-for-review paths are blocked from this environment (`_mark_pull_request_ready_for_review` GraphQL `htmlUrl` field bug; local `gh pr ready` returns HTTP 401 bad credentials).

Files changed in the final pushed branch:
- `dharma_swarm/operator_brief/__init__.py`
- `dharma_swarm/operator_brief/insight_brief.py`
- `dharma_swarm/cron_runner.py`
- `cron_jobs.json`
- `tests/test_operator_brief_insight_brief.py`

Tests run:
- `python -m pytest tests/test_operator_brief_insight_brief.py -v` — 14 passed
- `python -m pytest tests/test_telic_seam.py tests/test_telos_gates.py tests/test_ontology.py -q` — 145 passed
- `python -m pytest tests/test_cron_runner.py tests/test_cron_scheduler.py tests/test_cron_daemon.py -q` — 41 passed
- `python -m compileall -q dharma_swarm tests` — passed
- `git diff --check` — passed

Live tick:
- `DHARMA_OPERATOR_BRIEF_ENABLED=1 ... cron_run(...)` ran successfully once.
- `artifact_id=e4bccf8689cd4de5`
- `runtime_artifact_id=e4bccf8689cd4de5`
- artifact path and runtime `artifact_records.checksum` SHA-256 match.

Remaining risks:
- `memory_facts` support is wired and tested, but the current live runtime DB had zero `memory_facts`; the live tick cited `session_events` only.
- `STEELMAN` is still local marker-based enforcement until the upstream gate becomes semantically stronger.
- Failure ticks emit zero-value `ValueEvent`s for watchdog visibility; `Contribution` remains success-only by design.
- PR is code-ready but still needs the GitHub draft flag manually cleared.

Exact next step after merge: implement `NEXT_10_SUBSTRATE_TODO` item 8, the `LEDGER_WATCHER` check for operator-brief cron ticks that fail to produce `KnowledgeArtifact` rows.

**Comment by AmitabhainArunachala** [comment]
## Merge-conflict pass

Result: no current merge conflicts found against `main`.

Checks performed:
- Refreshed `origin/main` and `origin/feature/operator-brief-seam-v0`.
- Confirmed PR head is `e0cdb790c4ff020ede530ff34abd767b94d4abf2` and base is `1fca28d00b0d74bbd80782e5f1004772ed89c603`.
- Confirmed `origin/main` is an ancestor of the PR branch.
- Confirmed zero files changed on `main` after the merge-base.
- Simulated merge with `git merge-tree --write-tree origin/main HEAD`; it completed cleanly with no conflict markers/conflict exit.
- Ran `git diff --check` across the branch diff; clean.
- Re-ran targeted verification: `python -m pytest tests/test_operator_brief_insight_brief.py -q` — 14 passed; targeted `compileall` passed.

No conflict fix commit was needed. Remaining non-code blocker is unchanged: PR is still draft because the connector ready-for-review mutation fails on its GraphQL `htmlUrl` bug and local `gh` auth returns 401.

File: dharma_swarm/operator_brief/insight_brief.py [hunk 1]
```diff
@@ -0,0 +1,1217 @@
...
+    mode.
+    """
+
+    reg = registry if registry is not None else get_shared_registry()
+    keeper = gatekeeper or DEFAULT_GATEKEEPER
+    now_dt = now or datetime.now(timezone.utc)
+    date_str = now_dt.date().isoformat()
+    runtime = runtime_state
+    if runtime is None and registry is None and input_payload is None:
+        runtime = RuntimeStateStore()
```
[the remark this answers is not in the document]
Comment (AmitabhainArunachala) [reply dharma_swarm/operator_brief/insight_brief.py]: Addressed in `68d81d9`: the real cron/default runtime store path is now selected through `_default_runtime_state()` with an explicit `RuntimeStateStore(db_path=None)` call. The local hygiene gate passes and reports only the pre-existing `tests/test_full_loop.py` known warning.

**Comment by AmitabhainArunachala** [comment]
## PR #57 handoff

- verdict: ready from code/CI review; GitHub still shows Draft because the connector `markPullRequestReadyForReview` mutation fails on an `htmlUrl` schema bug and local `gh pr ready` is blocked by `HTTP 401: Bad credentials`.
- files changed: `cron_jobs.json`, `dharma_swarm/cron_runner.py`, `dharma_swarm/operator_brief/__init__.py`, `dharma_swarm/operator_brief/insight_brief.py`, `dharma_swarm/operator_brief/persistence.py`, `dharma_swarm/operator_brief/types.py`, `tests/test_operator_brief_insight_brief.py`.
- tests run locally: `python -m pytest tests/test_operator_brief_insight_brief.py -v`; `python -m pytest tests/test_telic_seam.py tests/test_telos_gates.py tests/test_ontology.py -q`; `python -m pytest tests/test_cron_runner.py tests/test_cron_scheduler.py tests/test_cron_daemon.py -q`; `python -m compileall -q dharma_swarm tests`; `python scripts/governance/check_module_budget.py --base-ref origin/main --head-ref HEAD`; `python scripts/governance/check_test_hygiene.py`.
- GitHub Actions on `f01c0bf`: tests, structure, test-hygiene, commit-lint, module-budget, gitleaks, semgrep, and CodeQL all passed.
- remaining risks: live runtime had no `memory_facts` rows, so the live tick cited `session_events`; `STEELMAN` is still local marker enforcement until the upstream gate grows stronger; `Contribution` is success-only; PR needs a manual “Ready for review” click or repaired GitHub auth before merge UI is fully ready.
- exact next step after merge: implement `NEXT_10_SUBSTRATE_TODO` item 8, the `LEDGER_WATCHER` check for operator-brief cron ticks that fail to produce `KnowledgeArtifact` rows.

**Comment by AmitabhainArunachala** [comment]
## PR #57 updated handoff after gate-block fix

- verdict: ready after blocker fix on `b66ffb9`.
- blocker addressed: `run_once()` no longer discards top-level `TelosGatekeeper` BLOCK decisions outside `CONSENT`, `BHED_GNAN`, `STEELMAN`, and `DOGMA_DRIFT`; non-required upstream blocks such as `AHIMSA`/`SATYA` are recorded as additional blocking gate decisions and prevent artifact materialisation.
- files changed: `cron_jobs.json`, `dharma_swarm/cron_runner.py`, `dharma_swarm/operator_brief/__init__.py`, `dharma_swarm/operator_brief/insight_brief.py`, `dharma_swarm/operator_brief/persistence.py`, `dharma_swarm/operator_brief/types.py`, `tests/test_operator_brief_insight_brief.py`.
- tests run locally: `python -m pytest tests/test_operator_brief_insight_brief.py -v` (`15 passed`); `python -m pytest tests/test_telic_seam.py tests/test_telos_gates.py tests/test_ontology.py -q` (`145 passed`); `python -m pytest tests/test_cron_runner.py tests/test_cron_scheduler.py tests/test_cron_daemon.py -q` (`41 passed`); `python -m compileall -q dharma_swarm tests`; `python scripts/governance/check_module_budget.py --base-ref origin/main --head-ref HEAD`; `python scripts/governance/check_test_hygiene.py`.
- GitHub Actions on `b66ffb9`: tests, structure, test-hygiene, commit-lint, module-budget, gitleaks, semgrep, and CodeQL all passed.
- remaining risks: live runtime had no `memory_facts` rows, so the live tick cited `session_events`; `STEELMAN` local enforcement is still marker-based until the upstream gate grows stronger; `Contribution` is success-only.
- exact next step after merge: implement `NEXT_10_SUBSTRATE_TODO` item 8, the `LEDGER_WATCHER` check for operator-brief cron ticks that fail to produce `KnowledgeArtifact` rows.
